data/json/items/vehicle/engineering.json

```diff
@@ -24,5 +24,18 @@
     "material": "steel",
     "symbol": "&",
     "color": "dark_gray"
+  },
+  {
+    "id": "pallet_lifter",
+    "type": "GENERIC",
+    "category": "veh_parts",
+    "name": "pallet lifter",
+    "description": "A makeshift pallet lifter.  If attached to a frame it could be used to lift up to 500 kg.",
```

Comment:
Suggest changing `500 kg` to `0.5 metric tonnes` to keep in line with other lifters.

Reply:
Done.